gtk4/src/actionable.rs

```diff
@@ -1,25 +1,22 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
 use crate::Actionable;
-use glib::object::IsA;
 use glib::translate::*;
-use libc::c_char;
+use glib::{IsA, ToVariant};
 
 // rustdoc-stripper-ignore-next
 /// Trait containing manually implemented methods of [`Actionable`](crate::Actionable).
 pub trait ActionableExtManual: 'static {
     #[doc(alias = "gtk_actionable_set_action_target")]
-    fn set_action_target(&self, string: &str);
+    fn set_action_target<V: ToVariant>(&self, string: &V);
 }
 
 impl<O: IsA<Actionable>> ActionableExtManual for O {
-    fn set_action_target(&self, string: &str) {
-        let string: Stash<*const c_char, _> = string.to_glib_none();
+    fn set_action_target<V: ToVariant>(&self, string: &V) {
```

Comment:
```suggestion
    fn set_action_target<V: ToVariant>(&self) {
```

You're not really using the value itself...